src/test/resources/apikey-test.properties

```diff
@@ -17,9 +17,9 @@ spring.mail.username =
 spring.mail.password =
 spring.mail.port = 0
 
-keycloak.authServerUrl           = https://localhost:8080/auth
+keycloak.authServerUrl           = https://keycloak-cf-test.eanadev.org/auth
```

Comment:
Please do not use test services. If these go down, your test will break. Unit tests should be able to run in isolation.